amulet/helpers.py

```diff
@@ -58,7 +58,16 @@ def run_bzr(args, working_dir, env=None):
     return _as_text(out)
 
 
-def juju(args, env=None):
+def juju(args, env=None, include_env=True):
```

Comment:
Can we use `include_model` or `with_model` env has a contest of environment variable and not juju env